src/projectscene/qml/Audacity/ProjectScene/clipsview/TracksClipsView.qml

```diff
@@ -204,6 +243,9 @@ Rectangle {
                 }
             }
             onReleased: e => {
+                if (e.mouse === Qt.RightButton) {
```

Comment:
better to check for e.mouse !== Qt.LeftButton, because there are also Middle and Back(dunno what is it) buttons